### ISSUE TITLE
Fallback fails

### ISSUE THREAD

**Issue by developdeez** [i18next/react-i18next#848]
I have my fallback set to 'en'. Some times my browser comes in as en-US. I was hoping it would fallback to 'en' since I dont have that folder but instead it failed:

![image](https://user-images.githubusercontent.com/13059525/57189641-c5f08e00-6ec5-11e9-8512-78bbcee0481b.png)

**Comment by jamuhl** [comment]
https://www.i18next.com/principles/fallback#language-fallback

if not set different it will try loading en-US and en.

You can play with load and whitelist option to further configure this

**Comment by clarencefoy** [comment]
I have the same issue. I've set `fallbackLng: 'en'`, but when the browser language is detected as 'en-US', the app just crashes without any useful error messages. I've tried various workarounds to no avail.

I am wondering whether this has to do with me defining the following in resources:

```
    resources: {
      en: {
        common: common_en, //(points at './translations/en/common.json' )
      },
      de: {
        common: common_de, //(points at './translations/de/common.json' )
      },
    },
```
I haven't defined a .json file for en-US, but I figured fallbackLng was specifically there for 'missing' languages.
So I suppose I could add every language code available there and default them all to 'common_en', but that seems inefficient?
